.vscode/settings.json

```diff
@@ -19,5 +19,6 @@
   },
   "python.formatting.provider": "none",
   "python.terminal.activateEnvironment": false,
-  "python.analysis.typeCheckingMode": "basic"
+  "python.analysis.typeCheckingMode": "basic",
+  "cSpell.words": ["reactflow"]
```

Comment:
explain? confused

Reply:
Sorry, this isn't meant to be uploaded. In fact, all of .vscode should be gitignored

Reply:
If you're wondering, I'm using Code Spell Checker and 'reactflow' isn't recognized as an English word, so I added it to my custom dictionary